### PR TITLE
Examples with typst

### PR DESCRIPTION
**Pull request by arnaudgolfouse**
Remove the custom runner, and use the `typst` binary instead, with assertions.

This really cements the idea that the protocol is made for typst, and we don't really care about general use cases. I believe this is a reasonable path to take (especially since this repo is linked from the [typst documentation](https://typst.app/docs/reference/foundations/plugin/) !), but tell me what you think !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,69 +20,28 @@ pub fn hello() -> Vec<u8> {
...
-emcc --no-entry -s ERROR_ON_UNDEFINED_SYMBOLS=0 -o hello.wasm hello.c
-```
+If you have all the required dependencies, you may build all examples by running `cargo test`.
 
-### Tips
+## wasi-stub
 
-- `host-wasmi` does not support running with WASI, and will stub all WASI functions instead.
-- host-wasmi compiles fastest 😉
+The runtime used by typst to not allow the plugin to import any function (beside the ones used by the protocol). In particular, if your plugin is compiled for [WASI](https://wasi.dev/), it will not be able to be loaded by typst.
```
Comment (astrale-sharp) [inline README.md:43]: do* not allow?

Comment (arnaudgolfouse) [reply README.md]: 👍

**Comment by astrale-sharp** [comment]
good to go?

**Comment by arnaudgolfouse** [comment]
Ya !

**Pull request merged**